File: backend/internal/meetings/model/meeting.go
package model

import "time"

type Meeting struct {
	ID             string
	BuildingID     string
	InitiatorName  string
	ScheduledAt    time.Time
	Location       string
	Agenda         []string
	MeetingForm    string
	CreatedBy      string
	CreatedAt      time.Time
	NotificationID *string
	AnnouncementID *string
}

File: frontend/src/types/meeting.ts
export type Meeting = {
  id: string;
  building_id?: string | null;
  initiator_name: string;
  scheduled_at: string;
  location: string;
  agenda: string[];
  meeting_form: string;
  notification_id?: string | null;
  announcement_id?: string | null;
  status: string;
};
